libs/ui/search/src/lib/record-preview-row/record-preview-row.component.html

```diff
@@ -20,7 +20,7 @@
   >
     <div class="mb-3 mt-5 sm:mt-2">
       <div
-        class="font-title text-21 text-title line-clamp-2 col-start-1 col-span-2 sm:line-clamp-1 group-hover:text-primary"
+        class="font-title text-21 text-title line-clamp-2 col-start-1 col-span-2 sm:line-clamp-1 group-hover:text-primary transition-colors"
```

Comment:
The transition has not been applied to the feed preview, but I guess this is on purpose (as it already has the shadow transition and primary org title).